docs/developing/integrate/neon_transfer.mdx

```diff
@@ -112,16 +113,16 @@ import {
 } from '@neonevm/token-transfer-core';
 
 const neonToken: SPLToken = {
-  chainId,
-  address_spl: '89dre8rZjLNft7HoupGiyxu3MNftR577ZYu8bHe2kK7g',
+  chainId: neonChainId,
+  address_spl: neonTokenMint.toBase58(),
   address: '',
   decimals: NEON_TOKEN_MINT_DECIMALS,
   name: 'Neon',
   symbol: 'NEON',
   logoURI: 'https://raw.githubusercontent.com/neonlabsorg/token-list/main/neon_token_md.png'
 };
 
-const transaction = await solanaNEONTransferTransaction(solanaWallet.publicKey, neonWallet.address, neonEvmProgram, neonTokenMint, neonToken, amount, chainId);
+const transaction = await solanaNEONTransferTransaction(solanaWallet, neonWallet, neonEvmProgram, neonTokenMint, neonToken, amount, chainId);
 const signature = await sendSolanaTransaction(connection, transaction, [toSigner(solanaWallet)]);
```

Comment:
Change this `const signature = await sendSolanaTransaction(connection, transaction, [toSigner(solanaWallet)]);` to:
```
const signers = [toSigner(solanaWallet)];
transaction.recentBlockhash = (await connection.getLatestBlockhash('finalized')).blockhash; // Network blockhash
transaction.sign(...signers);
const signature = await connection.sendRawTransaction(transaction.serialize());
```